Modix.Data/Repositories/MessageRepository.cs

```diff
@@ -339,6 +326,9 @@ public async Task UpdateStarboardColumn(ulong messageId, ulong? starboardEntryId
                 .Where(x => x.Id == messageId)
                 .FirstOrDefaultAsync();
 
+            if (entity is null)
```

Comment:
I don't think that this query realistically can fail, so could probably be changed into `.First()` instead?

Reply:
Sure, I think I'm fine with erroring with we don't find the message record